modules/azurerm/mx/hub.tf

```diff
@@ -0,0 +1,89 @@
+locals {
```

Comment:
You have many files that are duplicates from aws. I had this problem with sonar. But this is worst for DAM.
I think we should have a way to share common code between the 2. This effort might deserve a dedicate task

Reply:
I'll open a Jira task for it, it is relevant to all azure modules: Sonar, DAM, DRA

Reply:
https://onejira.imperva.com/browse/EDSF-524